src/test/shared/telemetry/telemetryUtils.test.ts

```diff
@@ -98,7 +93,6 @@ describe('telemetryUtils', () => {
                             assert.strictEqual(metadata.get('foo'), 'bar')
                             assert.strictEqual(metadata.get('hitcount'), '5')
 
-                            assert.strictEqual(mockService.lastEvent!.namespace, 'Command')
                             assert.strictEqual(data.name, 'somemetric')
```

Comment:
Wouldn't this be "Command.somemetric" ?

Reply:
wouldn't what be?

Reply:
This test may not be exercising the validation checks